libs/npm/core/src/components/text/Text.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { InputHTMLAttributes } from 'react';
```

Comment:
Added Text to tno-core so we could use it in our package.

Reply:
Follow same rules for location as our main library `/components/form/text`.
Also remove from our main library or we won't maintain them both.
Eventually all our form components will be moved over, but we really need to figure out how to test without publishing.

Reply:
Currently doing that, also realized I left in the old component. So updating it all to use the newly moved one!